app/models/free_training.rb

```diff
@@ -0,0 +1,2 @@
+class FreeTraining < ApplicationRecord
```

Comment:
Maybe it is a good idea to add a comment to this class explaining that it is read only. And the date served by this class is static.